<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Shivam" failures="1" tests="5" name="Test" time="142.942" errors="0" timestamp="2024-11-07T14:25:47 IST">
  <testcase classname="extender.Extender" name="TC_01_userLogin" time="30.178"/>
  <testcase classname="extender.Extender" name="TC_02_surveyDetailsSubmitting" time="13.29"/>
  <testcase classname="extender.Extender" name="TC_03_beforeCreditBalanceCheck" time="20.336"/>
  <testcase classname="extender.Extender" name="TC_04_toolUsingJourney" time="49.725"/>
  <testcase classname="extender.Extender" name="TC_05_afterCreditBalance" time="20.187">
    <failure type="java.lang.AssertionError" message="Credit did not increment as expected. expected [12] but found [10]">
      <![CDATA[java.lang.AssertionError: Credit did not increment as expected. expected [12] but found [10]
at extender.Extender.TC_05_afterCreditBalance(Extender.java:255)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- TC_05_afterCreditBalance -->
</testsuite> <!-- Test -->
